src/services/database.ts

```diff
@@ -65,6 +67,25 @@ export const searchByAll = async (searchIdentifier: string): Promise<SearchResul
   }
 };
 
+export const getBySystemNumberAndCreatedTimestamp = async (systemNumber: string, createdTimestamp: string): Promise<object> => {
+  const command: GetItemCommandInput = {
+    TableName: tableName,
+    Key: marshall({
+      systemNumber,
+      createdTimestamp,
+    }),
+  };
+
+  try {
+    const data = await ddbClient.send(new GetItemCommand(command));
+    logger.debug(JSON.stringify(data));
+    return unmarshall(data.Item || {});
+  } catch (e: any) {
+    logger.error('Error in search by sysnum and time: ', e);
```

Comment:
stringify the error? (also I'm not sure the logger library accepts multiple arguments to log)